hipaa-npi-number-input.html

```diff
@@ -105,6 +109,8 @@
           this.set('invalid', this.invalid);
         } else {
           this.$.container.invalid = !valid;
```

Comment:
@anushagadepalli 
This entire method needs to be rewritten. I think at a minimum u need something like the below code:
```
      validate: function() {
        if (this.$.input.validate() && this.value && this.value.length === 10) {
          this.invalid = !this._checkIfValid(); // jshint ignore:line
          this.$.container.invalid = this.invalid;
          this.set('invalid', this.invalid);
        } else {
          this.invalid = true;
          this.$.container.invalid = this.invalid;
          this.set('invalid', this.invalid);
        }
        return !this.invalid;
      }
```